Utils/github_workflow_scripts/handle_new_contribution.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+from demisto_sdk.commands.common.git_content_config import GitContentConfig, GitProvider
+from demisto_sdk.commands.common.tools import get_remote_file
+
+
+def main():
+    content_roles = get_remote_file(
+        ".github/content_roles.json",
+        git_content_config=GitContentConfig(
+            repo_name=GitContentConfig.OFFICIAL_CONTENT_REPO_NAME,
+            git_provider=GitProvider.GitHub,
+        ),
+    )
+    contrib_tl_username = content_roles["CONTRIBUTION_TL"]
+    if not contrib_tl_username:
+        print("No contribution TL")  # noqa: T201
+        sys.exit(1)
+    # save the contrib_tl username to a file for a later use in the workflow
+    print(f"{contrib_tl_username=}")  # noqa: T201
```

Comment:
If we are echoing the tl name in the handle_new_contribution.yml I think we can remove this print